src/pyflex/config.py

```diff
@@ -23,8 +23,10 @@ class Config(object):
     def __init__(self, min_period, max_period, stalta_waterlevel=0.07,
                  tshift_acceptance_level=10.0, tshift_reference=0.0,
                  dlna_acceptance_level=1.3, dlna_reference=0.0,
-                 cc_acceptance_level=0.7, s2n_limit=1.5, earth_model="ak135",
+                 cc_acceptance_level=0.7, earth_model="ak135",
+                 s2n_limit_energy=2.0, s2n_limit_amplitude=1.5,
```

Comment:
This is tricky because it will unexpectedly break existing code for people who use the old `s2n_limit`. After this PR is released I'll release a new major version to make it clear that something changed.

Can you please also add the following:

```python
def __init__(self, ..., **kwargs):
    if "s2n_limit" in kwargs:
        raise PyflexError(
            "The `s2n_limit` argument no longer exists. "
            "Please use `s2n_limit_energy` and/or `s2n_limit_amplitude`")
```

Then people will be aware that something changed and it raises a descriptive error with a clear message. I don't think we have to do a proper deprecation process for pyflex. Its too much work...